docs/4.tutorials-and-examples/6.custom-webchat.mdx

```diff
@@ -115,24 +167,109 @@ export const MyHeader = props => {
+      borderRadius: '25px',
+      boxShadow: '0 0 50px rgba(0,0,255,.30)',
+      backgroundImage:
+        'linear-gradient(to top, #ffffff,#ffffff 11%,#9a9ae3 40%,#0000ff 85%,#0000ff 85%)'
+    },
+    brandColor: '#0000ff',
+    textPlaceholder: 'Type something...',
+    botLogoChat: MyBotLogoChat,
+    headerTitle: 'My customized webchat',
+    headerSubtitle: 'R2D2',
+    headerImage: MyCustomHeaderImage,
+    introImage: MyCustomIntroImage,
+    customUserMessages: {
+      color: 'white',
+      borderRadius: '20px'
+    },
+    customBotMessages: {
+      color: 'blue',
+      borderRadius: '10px'
+    },
+    customReply: MyCustomReply,
+    alignReplies: 'right',
+    wrapReplies: 'wrap',
+    customButton: MyCustomButton,
+    brandIconUrl: launcherIcon,
+    customTriggerButton: TriggerCustom,
+    customHeader: MyCustomHeader
+  }
+}
+```
+
+All this changes can be tested using the `botonic serve` command *(`npm run start` on Windows)*. 
+Try to play with the properties in the template in order to see how they affect to the differents components in the webchat. 
+This is how it looks like.
+
+<img
+  src="https://botonic-doc-static.netlify.com/images/webchat_example.png"
+  style={{ display: 'block', margin: 'auto', maxWidth: '40%' }}
+/>
 
-import { routes } from './routes'
-import { locales } from './locales'
-import { integrations } from './integrations'
-import { theme } from './webchat'
+## Persistent Menu
+By adding these few lines of code in webchat's object, you can enable the persistent menu:
 
-export default new App({ routes, locales, integrations, theme })
+```javascript
+persistentMenu: [
+    { label: 'Help', payload: 'help' },
+    {
+      label: 'See docs',
+      url: 'https://docs.botonic.io'
+    }
+  ]
 ```
 
-All this changes it can be tested using the `botonic serve` command *(npm run start on windows)*. This is how it looks like.
+<img
+  src="https://botonic-doc-static.netlify.com/images/webchat_persistent_menu.png"
+  style={{ display: 'block', margin: 'auto', maxWidth: '50%' }}
+/>
+
+_Example of how it looks the Persistent Menu_
+
+The persistent menu lets you to use the same properties as a Botonic Button, this is `payload` and `url`.
+## Block Inputs
+Sometimes you would want to not allow certain messages for security reasons or to avoid agressive words. You can block these inputs as follows:
 
+```javascript
+blockInputs: [
+    {
+      match: [/ugly/, /bastard/],
+      message: 'We cannot tolerate these kind of words.'
+    }
+  ]
+```
 <img
-  src="https://botonic-doc-static.netlify.com/images/webchat_example.png"
-  style={{ display: 'block', margin: 'auto', maxWidth: '100%' }}
+  src="https://botonic-doc-static.netlify.com/images/webchat_block_inputs.png"
+  style={{ display: 'block', margin: 'auto', maxWidth: '25%' }}
 />
+
+_Example of blocking inputs_
+
+Once the specified inputs are matched, an automatic message will be displayed warning the user and the bot won't execute any internal logic.
```

Comment:
Once the specified inputs are matched:
* The use input will not be displayed in the message history
* The configured message will appear.
* The bot will not receive the blocked message